Test results with QuantLib 1.10

Here I share the results of pqyl+Quantlib1.10 `python setup.py test` run:

> Test failed: <unittest.runner.TextTestResult run=578 errors=2 failures=6>

Errors:

>     from .shortrate.onefactormodels.hullwhite import HullWHite
> ImportError: cannot import name 'HullWHite'

> FAIL: test_extrapolation (quantlib.test.test_zero_coupon.ZeroCouponTestCase)
> ----------------------------------------------------------------------
> Traceback (most recent call last):
>   File "/home/zvyagin/Projects/QL/pyql.git/quantlib/test/test_zero_coupon.py", line 108, in test_extrapolation
>     dtMax = ts.max_date
> AssertionError: RuntimeError not raised

> FAIL: test_zero_curve (quantlib.test.test_zero_coupon.ZeroCouponTestCase)
> ----------------------------------------------------------------------
> Traceback (most recent call last):
>   File "/home/zvyagin/Projects/QL/pyql.git/quantlib/test/test_zero_coupon.py", line 139, in test_zero_curve
>     self.assertAlmostEqual(zr.rate, 0.0539332)
> AssertionError: 0.05393326083960954 != 0.0539332 within 7 places

--- a/quantlib/models/api.py
+++ b/quantlib/models/api.py
@@ -2,4 +2,4 @@ from .equity.heston_model import HestonModelHelper, HestonModel
 from .equity.bates_model import (BatesModel, BatesDetJumpModel, BatesDoubleExpModel,
     BatesDoubleExpDetJumpModel)
 from .calibration_helper import (RelativePriceError, PriceError, ImpliedVolError)
-from .shortrate.onefactormodels.hullwhite import HullWHite
+from .shortrate.onefactormodels.hullwhite import HullWhite

--- a/quantlib/test/test_zero_coupon.py
+++ b/quantlib/test/test_zero_coupon.py
@@ -103,9 +103,11 @@ class ZeroCouponTestCase(unittest.TestCase):
 
         # max_date raises an exception without extrapolaiton...
         self.assertFalse(ts.extrapolation)
-        with self.assertRaisesRegexp(RuntimeError,
-                                     "1st iteration: failed at 2nd alive instrument"):
-            dtMax = ts.max_date
+        with self.assertRaises(RuntimeError) as ctx:
+            ts.discount(ts.max_date + 1)
+        self.assertTrue(str(ctx.exception) in (
+            "time (30.011) is past max curve time (30.0082)",
+            "1st iteration: failed at 2nd alive instrument"))
 
     def test_zero_curve(self):
         rate_helpers = build_helpers()
@@ -136,7 +138,7 @@ class ZeroCouponTestCase(unittest.TestCase):
         # max_date raises an exception...
         ts.extrapolation = True
         zr = ts.zero_rate(Date(10, 5, 2027), ts_day_counter, 2)
-        self.assertAlmostEqual(zr.rate, 0.0539332)
+        self.assertAlmostEqual(zr.rate, 0.0539332, 6)
 
 if __name__ == '__main__':
     unittest.main()

